src/components/BulkEnrollmentPage/ConnectedPagination.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {
+  useEffect, useContext,
+} from 'react';
+import PropTypes from 'prop-types';
+import { connectPagination } from 'react-instantsearch-dom';
+import {
+  DataTable, DataTableContext,
+} from '@edx/paragon';
+
+export const BaseConnectedPagination = ({
+  currentRefinement: algoliaPageNo,
+  refine,
+}) => {
+  const { state: tableState } = useContext(DataTableContext);
+  useEffect(() => {
+    if (algoliaPageNo && tableState.pageIndex + 1 !== algoliaPageNo) {
```

Comment:
nit: is it against style guidelines to use an extra set of parentheses here?

Reply:
No, it's not, and I think we should.